src/cable/connection.cr

```diff
@@ -153,5 +159,23 @@ module Cable
     def self.broadcast_to(channel : String, message : String)
       Cable.server.publish(channel, message)
     end
+
+    private def internal_channel
+      "cable_internal/#{internal_identifier}"
+    end
```

Comment:
If you don't use `identified_by`, and have multiple connections, my guess is bad things might happen :man_shrugging: